packages/compiler-cli/src/ngtsc/typecheck/src/context.ts

```diff
@@ -128,6 +128,22 @@ export class TypeCheckContext {
    */
   private typeCtorPending = new Set<ts.ClassDeclaration>();
 
+  /**
+   * Map of data for file paths which was adopted from a prior compilation.
```

Comment:
Given that it's not the map itself but its entries that are adopted, this should be:

```suggestion
   * Map of data for file paths which were adopted from a prior compilation.
```